lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -37,6 +37,10 @@ html.video-fullscreen{
       color: $white;
     }
   }
+
+  @media print {
```

Comment:
This ensures that instructor buttons such as "View in Studio", "Staff Debug Info", and "Student Submission History" do not print.